### PR TITLE
adding nsls2-detector-handlers recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: https://github.com/nsls-ii-forge/nsls2-detector-handlers-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

File: recipes/nsls2-detector-handlers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+test:
+  imports:
+    - nsls2_detector_handlers
+
+about:
+  home: https://github.com/bluesky/nsls2-detector-handlers
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: Staging repo for handlers specific to NSLS-II
```
Comment (dopplershift) [inline recipes/nsls2-detector-handlers/meta.yaml:37]: That doesn't sound like a great description for the *package*

**Comment by dopplershift** [comment]
Recipe otherwise looks good pending confirmation from @mrakitin

File: recipes/nsls2-detector-handlers/meta.yaml (hunk 1 above)
Comment (mrakitin) [inline recipes/nsls2-detector-handlers/meta.yaml:37]:
```suggestion
  summary: Detector handlers specific to NSLS-II
```

Comment (mrakitin) [reply recipes/nsls2-detector-handlers/meta.yaml:37]: I agree. I suggested a slight modification so it's a bit more self-explanatory.

**Comment by mrakitin** [comment]
> Recipe otherwise looks good pending confirmation from @mrakitin

I agree to maintain this recipe/feedstock.

**Review by dopplershift — APPROVED** [review]

**Comment by mrakitin** [comment]
Thanks, Ryan!

**Pull request merged**